Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -121,21 +121,22 @@ namespace Fuse.Layouts
 			if (_hasItemHeight)
 				clp.SetY(ItemHeight);
 
-			var placements = new float4[elements.Count];
+			var placements = new float4[elements.Children.Count];
```

Comment:
My idea is to refactor this so only the Visual.Children.LinkedList.uno file uses the weird `Children_*` symbols. All other files should use a "proper" interface